src/util/user-home-dir.js

```diff
@@ -4,7 +4,8 @@ import ROOT_USER from './root-user.js';
 
 const path = require('path');
 
-const userHomeDir =
-  process.platform === 'linux' && ROOT_USER ? path.resolve('/usr/local/share') : require('os').homedir();
+export const home = require('os').homedir();
+
+const userHomeDir = process.platform === 'linux' && ROOT_USER ? path.resolve('/usr/local/share') : home;
```

Comment:
It seems safe to add `'freebsd'` to the list, are there any other platforms we should add from here? https://nodejs.org/api/process.html#process_process_platform